build-system/tasks/json-check.js

```diff
@@ -21,6 +21,33 @@ const log = require('fancy-log');
 const through2 = require('through2');
 const {jsonGlobs} = require('../config');
 
+const expectedCaches = ['cloudflare', 'google'];
+
+/**
+ * Fail if caches.json is missing some expected caches.
+ */
+function checkCachesJson() {
+  return gulp.src(['caches.json',])
+      .pipe(through2.obj(function(file) {
+        try {
+          const obj = JSON.parse(file.contents.toString());
+        } catch (e) {
+          // Do nothing since this is handled in checkValidJson.
+        }
+        let foundCaches = [];
+        for (var i = 0; i < obj.caches.length; i++) {
```

Comment:
You're already using the for..of syntax below, so why not do this instead?
```javascript
for (const foundCache of obj.caches) {
  foundCaches.push(foundCache.id);
}
```

Looks cleaner :)